### PR TITLE
Add CMIP6 files

### PR DESCRIPTION
**Pull request by abarciauskas-bgse**
## Content checklist

Check the [Content Documentation](https://github.com/NASA-IMPACT/veda-ui/blob/main/docs/CONTENT.md) for more details.

### Any content
- [x] Cover image with proper attribution (when relevant) is present.
- [x] Cover images do not contain any text (they're used as a background and text may not be readable).
- [x] All images are not too big/heavy. As a general rule of thumb, they should not weigh more than 500KB, or be larger than 2000px. See the [media guide](https://github.com/NASA-IMPACT/veda-ui/blob/main/docs/frontmatter/media.md#media) for more info.
- [x] Content was added using the available [MDX Blocks](https://github.com/NASA-IMPACT/veda-ui/blob/main/docs/MDX_BLOCKS.md).

### Datasets
- [x] [Layer description](https://github.com/NASA-IMPACT/veda-ui/blob/main/docs/frontmatter/layer.md#properties) is defined.
- [x] [Layer legend](https://github.com/NASA-IMPACT/veda-ui/blob/main/docs/frontmatter/layer.md#legend) is setup.
- [x] The Dataset is showing up on the map correctly

### CODE REVIEW

**Comment by abarciauskas-bgse** [comment]
@slesaad @j08lue @danielfdsilva @hanbyul-here  I'm seeing an error from the deploy preview: https://deploy-preview-323--visex.netlify.app/data-catalog/combined_CMIP6_daily_GISS-E2-1-G_tas_kerchunk_DEMO/explore

>Warning: Received `true` for a non-boolean attribute `on`.
>
>If you want to write it to the DOM, pass a string instead: on="true" or on={value.toString()}.
article

which I thought may driven by something in the MDX file so I changed a few things but it hasn't resolved the issue. Do you have any pointers for debugging this? I'm not seeing it locally with veda-ui

Locally I can see this traceback:

```
T@https://deploy-preview-323--visex.netlify.app/index.fcca913e.js:14:286605
T@https://deploy-preview-323--visex.netlify.app/index.fcca913e.js:14:286605
u<@https://deploy-preview-323--visex.netlify.app/s-explore.a502d860.js:123:93
b/<@https://deploy-preview-323--visex.netlify.app/index.fcca913e.js:33:1837
x/<@https://deploy-preview-323--visex.netlify.app/index.fcca913e.js:33:6463
x@https://deploy-preview-323--visex.netlify.app/s-explore.a502d860.js:109:68
li
ol
T@https://deploy-preview-323--visex.netlify.app/index.fcca913e.js:14:286605
g@https://deploy-preview-323--visex.netlify.app/mdx-content.2e230bff.js:178:4077
h@https://deploy-preview-323--visex.netlify.app/s-explore.a502d860.js:62:79
div
```

**Comment by abarciauskas-bgse** [comment]
@anayeaye pointed me to codespaces https://github.com/NASA-IMPACT/veda-config/blob/develop/docs/GH_CODESPACES.md so I will try troubleshooting there if no one has any other ideas as to what could be going wrong.

**Comment by slesaad** [comment]
@abarciauskas-bgse took me a while to figure out 😅 but the `.env` is missing the titiler xarray endpoint

File: datasets/cmip6-tas.data.mdx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      variable: tas
+      colormap_name: coolwarm
+      rescale: 232,312
+      maxzoom: 12
+    legend:
+      unit: 
+        label: 
+      type: gradient
+      min: "232 K"
+      max: "312 K"
```
Comment (slesaad) [inline datasets/cmip6-tas.data.mdx:37]:
```suggestion
        label: K
      type: gradient
      min: 232
      max: 312
```

**Comment by hanbyul-here** [comment]
@abarciauskas-bgse I can confirm that this error was not introduced by the change of this PR. We have been aware of the error but have been pushing off because of the other priorities. so please ignore it for now. I will ticket this and handle.

**Comment by abarciauskas-bgse** [comment]
@slesaad 🙇🏽‍♀️ thank you so much

how did you track it down?

I believe this PR is now good to go. I slightly modified the text in the last section of the dataset description (see https://github.com/NASA-IMPACT/veda-config/pull/323/commits/244fad32abe288548b39a135384a3182eb863d4b) so let me know if that content looks good to you @slesaad @j08lue

**Review by j08lue — APPROVED** [review]
Looks great, @abarciauskas-bgse!

**Comment by slesaad** [comment]
@abarciauskas-bgse I noticed that the network request wasn't being made at all

also noticed that the provider URL in the collection metadata is "https://github.com/nasa-impact/veda-data-pipelines". We're no longer using that repository, so I'm kinda hesitant about putting that there.

But this PR looks good!

**Review by slesaad — APPROVED** [review]

**Pull request merged**